src/Umbraco.Core/Compose/AuditEventsComponent.cs

```diff
@@ -49,9 +49,8 @@ private IUser CurrentPerformingUser
             get
             {
                 var identity = Thread.CurrentPrincipal?.GetUmbracoIdentity();
-                return identity == null
-                    ? new User { Id = 0, Name = "SYSTEM", Email = "" }
-                    : _userService.GetUserById(Convert.ToInt32(identity.Id));
+                var user = identity == null ? null : _userService.GetUserById(Convert.ToInt32(identity.Id));
+                return user ?? new User { Id = 0, Name = "SYSTEM", Email = "" };
```

Comment:
Please use `Constants.Security.UnknownUserId` instead of `Id = 0`

Reply:
I also think the "SYSTEM" name should be in a constant, we use it multiple times

Reply:
Agreed, fixing.